### DISCUSSION TITLE
Hold press button in Apple TV

### DISCUSSION THREAD

**Discussion by santtyamen** [appium/appium#15332] [General]
Hello, I am automated on Apple TV devices, I need to press and hold the play button

$driver.execute_script('mobile: pressButton', { name: "playPause" })

But I think another parameter "hold press" is needed

$driver.execute_script('mobile: pressButton', { name: "playPause", holdPress: 5 })

Any ideas??

**Comment by KazuCocoa** [comment]
No for now, but https://developer.apple.com/documentation/xctest/xcuiremote/1627475-pressbutton may help.
Let me add the option.

**Reply by santtyamen** [reply]
It is a very important feature that can help in automation on Apple TV.

**Reply by KazuCocoa** [reply]
I've added `durationSeconds` https://github.com/appium/appium-xcuitest-driver#mobile-pressbutton
It is available in `appium@beta`

**Reply by santtyamen** [reply]
It is Wonderfull! I will wait the release with this feature!!

**Reply by Edwick-Raj** [reply]
@santtyamen hey are u able to use this feature? I've tried to implement fastforward in automation by press and holding the right button... But for me was not working... If possible could u please share the code snippet u have used for long press. 🙂
